products/session_summaries/manifest.tsx

```diff
@@ -0,0 +1,30 @@
+import { ProductManifest } from '../../frontend/src/types'
+
+export const manifest: ProductManifest = {
+    name: 'Session summaries',
+    scenes: {
+        SessionGroupSummariesTable: {
+            name: 'Session summaries',
+            import: () => import('./frontend/SessionGroupSummariesTable'),
+            projectBased: true,
+            description:
+                'View and deep-dive into AI-generated summaries of session recordings. Create summaries from the Session replay page by applying filters and asking PostHog AI to summarize sessions. If no filters, ask PostHog AI to find and summarize sessions based on provided criteria.',
```

Comment:
Three sentences are a lot here, I think we can skip the last one for now. Such descriptions are best when they're just imperative without conditionals